src/frontend/src/session.rs

```diff
@@ -370,7 +370,7 @@ impl SessionImpl {
             env,
             auth_context,
             user_authenticator,
-            config_map: RwLock::new(Default::default()),
+            config_map: RwLock::new(ConfigMap::new()),
```

Comment:
This suggests `ConfigMap::default()` is no longer safe to use and would produce an invalid config. Shall we impl it manually rather than derive, or use a custom type instead of `ConfigString`?

https://github.com/risingwavelabs/risingwave/blob/5e944c4f72f837efe99119116e4db45efc6ede67/src/common/src/session_config/mod.rs#L251-L252

Usage of `&'static str` as const generic is behind `#![feature(adt_const_params)]`